src/main/config/codelist/local/thesauri/theme/GC_Resource_Formats.rdf

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+         xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
          xmlns:skos="http://www.w3.org/2004/02/skos/core#"
```

Comment:
skos and ns2 are the same namespace. 
As FGP use ns2 - I suggest that you remove skos namespace and replace all occurrences with ns2. i.e. change `<skos:ConceptScheme>` to `<ns2:ConceptScheme>`